Source/RxSwift/filterMap.swift

```diff
@@ -22,6 +22,7 @@ extension ObservableType {
         - returning `.ignore` will filter the value out of the returned Observable
         - returning `.map(newValue)` will propagate newValue through the returned Observable.
      */
+    @available(*, deprecated, renamed: "compactMap", message: "compactMap was added to the main RxSwift repository and should be used instead of filterMap")
```

Comment:
```suggestion
    @available(*, deprecated, renamed: "compactMap", message: "Use compactMap instead")
```

I think this is enough for a deprecation message